experimenter/experimenter/changelog/views.py

```diff
@@ -1 +1,27 @@
-# Create your views here.
+from django.shortcuts import get_object_or_404
+from django.views.generic import DetailView
+
+from experimenter.experiments.models import NimbusExperiment
+
+
+class NimbusChangeLogsView(DetailView):
+    model = NimbusExperiment
+    template_name = "changelog/overview.html"
+    context_object_name = "experiment"
+
+    def get_object(self, queryset=None):
+        slug = self.kwargs.get("slug")
+        experiment = get_object_or_404(NimbusExperiment, slug=slug)
+        return experiment
+
+    def get_context_data(self, **kwargs):
+        slug = self.kwargs.get("slug")
+        context = super().get_context_data(**kwargs)
+        experiment = self.get_object()
+
+        changelogs = list(experiment.changes.all())
+
+        context["slug"] = slug
```

Comment:
Unnecessary. We have the experiment, so we can use `experiment.slug` in the template (and by default `kwargs` gets inserted into the context [here](https://github.com/django/django/blob/ddb6506618ea52c6b20e97eefad03ed847a1e3de/django/views/generic/detail.py#L100)